flocker/node/_loop.py

```diff
@@ -312,6 +312,13 @@ def sleep(self):
         :return _Sleep: an instance of `_Sleep` with a duration
             following an exponential backoff curve.
         """
+        Message.log(
+            message_type=u'flocker:node:_loop:delay',
+            log_level=u'INFO',
+            message=u'Intentionally delaying the next iteration of the '
+                    u'convergence loop to avoid RequestLimitExceeded.',
+            current_wait=self._delay
+        )
```

Comment:
The ``_Sleep`` object can't be serialized by Eliot.

 * http://ci-live.clusterhq.com:8080/job/ClusterHQ-flocker/job/enable-gce-driver-and-acceptance-tests-FLOC-3826/job/_run_acceptance_on_GCE_Ubuntu_Trusty_with_GCE/31/artifact/remote_logs.log
```
{"_PROCESS_NAME": "flocker-container-agent", "timestamp": 1457971496.562375, "task_uuid": "133f174a-94a8-48e3-b5ed-aabea78004e6", "_HOSTNAME": "104.197.90.163", "error": true, "message": "Unhandled Error\nTraceback (most recent call last):\n  File \"/usr/lib/python2.7/threading.py\", line 763, in run\n    self.__target(*self.__args, **self.__kwargs)\n  File \"/opt/flocker/local/lib/python2.7/site-packages/eliot/logwriter.py\", line 95, in _writer\n    self._reactor.run(installSignalHandlers=False)\n  File \"/opt/flocker/local/lib/python2.7/site-packages/twisted/internet/base.py\", line 1194, in run\n    self.mainLoop()\n  File \"/opt/flocker/local/lib/python2.7/site-packages/twisted/internet/base.py\", line 1203, in mainLoop\n    self.runUntilCurrent()\n--- <exception caught here> ---\n  File \"/opt/flocker/local/lib/python2.7/site-packages/twisted/internet/base.py\", line 798, in runUntilCurrent\n    f(*a, **kw)\n  File \"/opt/flocker/local/lib/python2.7/site-packages/eliot/_output.py\", line 384, in __call__\n    self.file.write(self._dumps(message) + self._linebreak)\n  File \"/usr/lib/python2.7/json/__init__.py\", line 243, in dumps\n    return _default_encoder.encode(obj)\n  File \"/usr/lib/python2.7/json/encoder.py\", line 207, in encode\n    chunks = self.iterencode(o, _one_shot=True)\n  File \"/usr/lib/python2.7/json/encoder.py\", line 270, in iterencode\n    return _iterencode(o, 0)\n  File \"/usr/lib/python2.7/json/encoder.py\", line 184, in default\n    raise TypeError(repr(o) + \" is not JSON serializable\")\nexceptions.TypeError: <_Sleep(delay_seconds=1.1977587997347745)> is not JSON serializable\n", "message_type": "twisted:log", "task_level": [1]}
```